.github/workflows/ci.yml

```diff
@@ -159,6 +159,8 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: write
+    # Only run on the main repository because forks do not have write access to the main repository.
```

Comment:
There is a lot more discussion here https://github.com/gradle/actions/blob/main/docs/dependency-submission.md#usage-with-pull-requests-from-public-forked-repositories
I'm not sure what we lose by disabling this here compared to doing the three separate jobs mentioned in that explanation. Can you take a look?

Reply:
I updated with three separate workflow. If we just disable it, there might be security issue in the PR and get merged into our repo.